ext/intl/grapheme/grapheme_string.c

```diff
@@ -125,8 +125,8 @@ PHP_FUNCTION(grapheme_strpos)
 	/* the offset is 'grapheme count offset' so it still might be invalid - we'll check it later */
 
 	if (needle_len == 0) {
-		intl_error_set( NULL, U_ILLEGAL_ARGUMENT_ERROR, "grapheme_strpos: Empty delimiter", 1 );
-		RETURN_FALSE;
+		zend_argument_value_error(2, "cannot be empty");
```

Comment:
Hmm, there is no such warning in `strpos()`, so maybe we could get rid of this error? On the other hand, the offset is not contained... case has been promoted to an exception in `strpos()`, so I'll do it here as well.

Reply:
Yes, we should add support for empty string here.